EIPS/eip-7697.md

```diff
@@ -0,0 +1,155 @@
+    5. Execution - any execution logic.
+
+[EIP-7697](./eip-7697.md) will address these issues.
+
+| Feature                                                      | [EIP-7697](./eip-7697.md)  | [EIP-3074](./eip-3074.md) | [EIP-5003](./eip-5003.md) | [EIP-7377](./eip-7377.md) |
+|--------------------------------------------------------------|----------------------------|---------------------------|---------------------------|---------------------------|
+| Deploy the same address [ERC-2470](./eip-2470.md)            | ✅                          | ❎                         | ✅ need 3 opcodes          | ✅ only EOA                |
+| Help EOA to upgrade to CA                                    | ✅                          | ❎                         | ✅ need 3 opcodes          | ✅                         |
+| Support `secp256r1` or more                                  | ✅                          | ✅                         | ❎                         | ❎                         |
+| Reuse existing wallet infrastructure                         | ✅ adapt the contract       | ❎                         | ✅ adapt the contract      | ❎ adapt the node rpc      |
+| Integrate easily with [ERC-4337](./eip-4337.md) and RIP-7560 | ✅ support factory contract | ❎                         | ❎                         | ❎                         |
+| Grant temporary CA capabilities to EOA                       | ❎                          | ✅                         | ❎                         | ❎                         |
+
+## Specification
+
+### Constants
+
+| Constant          | Value  |
+|-------------------|--------|
+| `AuthCreateMagic` | `0x04` |
+| `AUTHCREATE`      | `0xf6` |
+
+`AuthCreateMagic` is used for [EIP-7697](./eip-7697.md) signatures to prevent signature collisions with other signing
+formats.
+
+#### Input
+
+##### Stack
+
+| Stack     | Value       |
+|-----------|-------------|
+| `top - 0` | `endowment` |
+| `top - 1` | `offset`    |
+| `top - 2` | `size`      |
+
+##### Memory
+
+The two stack arguments (`offset` and `size`) describe a range of memory. The format of the contents of that range is:
+
+- `memory[offset    : offset+1 ]` - `yParity`
+    - 0-1: secp256k1
+    - 4-5: secp256r1
+- `memory[offset+1  : offset+33]` - `r`
+- `memory[offset+33 : offset+65]` - `s`
+- `memory[offset+65 :          ]` - `creationCode`
+
+#### Output
+
+##### Stack
+
+| Stack     | Value          |
+|-----------|----------------|
+| `top - 0` | `contractAddr` |
+
+##### Memory
+
+Memory is not modified by this instruction.
+
+#### Behavior
+
+The arguments (`yParity`, `r`, `s`) are interpreted as an ECDSA signature on the secp256k1(or secp256r1) curve over the
+message `keccak256(AuthCreateMagic || chainId || creationCodeHash)`, where:
+
+- `chainId` is the current chain's [EIP-155](./eip-155.md) unique identifier padded to 32 bytes.
+- `creationCode` is the creation code set for the EOA.
+
+#### Gas Cost
+
+_todo: detail_
```

Comment:
```suggestion
_todo: detail_ <!-- TODO -->
```

If you use HTML-style comments, the linter will make sure you come back and replace them.